### PR TITLE
Rename Alephant Cache to Alephant Storage

### PR DESCRIPTION
**Pull request by samfrench**
Rename Alephant Cache to Alephant Storage

### CODE REVIEW

**Comment by samfrench** [comment]
@BBCMarkMcDonnell, I now have the first Alephant Storage PR. I copied the code to here, but also thought it best to rename the cache references to storage.

I also set the version to 1.0.0, as I thought if approved, this would be the first version.

What do you think?

**Comment by samfrench** [comment]
I notice this repo isn't on Travis CI, so I don't know what I need to do to get it on there: https://travis-ci.org/BBC-News/

Will it appear automatically?

**Comment by Integralist** [comment]
Hi @samfrench - I'll get this added (I've signed into Travis CI using GitHub oauth and it's starting to sync up all public repos, but that could take a while). I'm not sure which account - probably Kenny's - that originally added the other Alephant gems to Travis.

**Comment by samfrench** [comment]
@BBCMarkMcDonnell, do I need to push this up again, for this to be picked up with Travis, or are you taking care of it?

**Comment by BBCMarkMcDonnell** [comment]
@samfrench re: Travis -> looks like it's there on Travis CI https://travis-ci.org/BBC-News/alephant-storage. It's setup to build on pushes and on pull requests, but it may be that there is some other setup required. I'll have a look through at the repo settings, but I'm not overly worried about it (for now) though.

**Comment by BBCMarkMcDonnell** [comment]
@samfrench I would also suggest changing this email https://github.com/BBC-News/alephant-storage/blob/master/.travis.yml#L7 to `DigitalNewsFrameworksTeam@bbc.co.uk`

File: Gemfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
 source 'https://rubygems.org'
 
-# Specify your gem's dependencies in alephant-cache.gemspec
+# Specify your gem's dependencies in alephant-storage.gemspec
```
Comment (BBCMarkMcDonnell) [inline Gemfile]: I'd just remove this line altogether to be honest

File: README.md [hunk 2]
```diff
@@ -19,23 +19,23 @@ And then execute:
...
+storage.put('id', "string data")
+storage.get('id')
 
 # => "string data"
 ```
 
 ## Contributing
 
-1. Fork it ( http://github.com/<my-github-username>/alephant-cache/fork )
+1. Fork it ( http://github.com/<my-github-username>/alephant-storage/fork )
```
Comment (BBCMarkMcDonnell) [inline README.md]: Change `<my-github-username>` to `BBC-News`

**Comment by BBCMarkMcDonnell** [comment]
Other than a few minor comments that I'd like to see made first, this is OK to :cake:

**Pull request merged**